ludwig/decoders/sequence_decoders.py

```diff
@@ -111,29 +108,21 @@ def __init__(
         #     state_size,
```

Comment:
I think it's useful to keep this code around since sequence decoders still need to be ported, and having the commented tf code can act as a guide for that.